live-build/misc/ansible-roles/appliance-build.minimal-common/tasks/main.yml

```diff
@@ -372,13 +372,24 @@
   # we need to rebuild the initramfs to account for the modprobe.d change.
   notify: update initramfs
 
-# Configure SSH to allow PAM "conversations" (interactions with the user).
 - lineinfile:
     path: /etc/ssh/sshd_config
     regexp: "^#?{{ item.key }}="
     line: "{{ item.key }}={{ item.value }}"
   with_items:
+    #
+    # Configure SSH to allow PAM "conversations" (interactions with the user).
+    #
     - { key: "ChallengeResponseAuthentication", value: "yes" }
+    #
+    # Harden the appliance by disabling ssh-agent(1), tcp, UNIX domain, and
+    # X11 forwarding. Note that this doesn't improve security unless users are
+    # also denied shell access.
+    #
+    - { key: "AllowAgentForwarding", value: "no" }
+    - { key: "AllowTcpForwarding", value: "no" }
+    - { key: "AllowStreamLocalForwarding", value: "no" }
```

Comment:
[nit] Can we please keep this list alphabetized? Keeping lists alphabetized when the order doesn't matter makes it easier to figure out where to place new lines in future changes.